### DISCUSSION TITLE
Homeassistant-core Update Error

### DISCUSSION THREAD

**Discussion by fumsi** [tteck/Proxmox#3836] [Q&A] [answered]
Hi, 

when i try to update homeassistant-core to the latest version i get this error

Updating to Stable Version
 ✓ Stopped Home Assistant
 / Updating Home Assistant   
[ERROR] in line 88: exit code 0: while executing command uv pip install ${BR}--upgrade homeassistant &> /dev/null

regards
ralf

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/issues/3817

https://github.com/tteck/Proxmox/blob/main/CHANGELOG.md#2024-10-03

**Comment by fumsi** [comment, marked as answer]
thx. reading https://github.com/tteck/Proxmox/issues/3817 fixed it.

**Comment by LocoR9** [comment]
I still have the same problem.

**Reply by tteck** [reply]
When all else fails, https://github.com/tteck/Proxmox/issues/3817#issuecomment-2391128551
